tqdm/tests/tests_tqdm.py

```diff
@@ -116,8 +116,11 @@ def write(self, s):
         self.text += s
         self.cursor = len(self.text)
```

Comment:
No no I meant what I wrote, because else if you seek and write, then the
cursor will always be placed at the end of the file instead of the position
after the just written characters. But indeed the change you propose should
also be included!

In fact it should be this to also not forget what was after:

self.text = self.text[:self.cursor] + s + self.text[self.cursor+len(s):]


2016-05-14 22:28 GMT+02:00 Casper da Costa-Luis <notifications@github.com>:

> In tqdm/tests/tests_tqdm.py
> <https://github.com/tqdm/tqdm/pull/162#discussion_r63281545>:
>
> > @@ -116,8 +116,11 @@ def write(self, s):
> >          self.text += s
> >          self.cursor = len(self.text)
>
> don't you mean self.text += s to self.text = self.text[:self.cursor] + s?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/tqdm/tqdm/pull/162/files/2a576c6f70d03e91ef1f41d95677ed882b36d354#r63281545>
>